signify/src/main.rs

```diff
@@ -20,50 +20,37 @@ use clap::{IntoApp, Parser};
     signify -V [-e] [-x <sigfile>] -p <pubkey> -m <message>"#
 )]
 struct Args {
-    #[clap(short = 'G', help = "Generate a new keypair.")]
+    /// Generate a new keypair
+    #[clap(short = 'G')]
     generate: bool,
-    #[clap(short = 'S', help = "Sign the specified message file.")]
+    /// Sign the specified message file
+    #[clap(short = 'S')]
     sign: bool,
-    #[clap(short = 'V', help = "Verify a message.")]
+    /// Verify a message
+    #[clap(short = 'V')]
     verify: bool,
-
-    #[clap(
-        short = 'p',
-        help = "Public key produced by -G, and used by -V to check a signature."
-    )]
+    /// Public key produced by -G, and used by -V to check a signature
+    #[clap(short, parse(from_os_str))]
     pubkey: Option<PathBuf>,
-
-    #[clap(
-        short = 's',
-        help = "Secret (private) key produced by -G, and used by -S to sign a message."
-    )]
+    /// Secret (private) key produced by -G, and used by -S to sign a message
+    #[clap(short, parse(from_os_str))]
     seckey: Option<PathBuf>,
-
-    #[clap(
-        short = 'n',
-        help = "Do not ask for a passphrase during key generation. Otherwise, signify will prompt the user for a passphrase to protect the secret key."
-    )]
+    /// Do not ask for a passphrase during key generation. Otherwise, signify
+    /// will prompt the user for a passphrase to protect the secret key
+    #[clap(short = 'n')]
     skip_key_encryption: bool,
-
-    #[clap(short = 'm')]
-    message_path: Option<String>,
-
-    #[clap(
-        short = 'e',
-        help = "When signing, embed the message after the signature. When verifying, extract the message from the signature."
-    )]
+    /// The file containing the message to create a signature over
```

Comment:
The initial part here is good but lets expand it to cover verifying both detached signatures and ~~embedded ones.~~ (Nevermind, I think this needs to be made way more clear in the CLI arguments anyway first).

What do you think about this:
```suggestion
    /// The file containing the message to create a signature over or to the one to verify with an existing signature
```